mflowgen/full_chip/dragonphy/outputs/dragonphy_RDL.gds

```diff
@@ -1 +1 @@
-/home/ajcars/dragonphy_top/dragonphy_RDL.gds
\ No newline at end of file
+/sim/steveri/dragonphy/Dec_2020/dragonphy_RDL.gds
```

Comment:
These should be in /home, not /sim so that the fullchip flow works on both arm and arm2 without maintaining separate versions.

Reply:
Yes, of course. Would you like to copy them to /home/ajcars so as to preserve the existing convention(s)? Then we can change the links

Reply:
just copied

Reply:
Updated the dragonphy links, they once again point to /home/ajcars instead of /sim/steveri, so this task is done.